### PR TITLE
[13.0] [MIG] purchase_reception_status

### PR DESCRIPTION
**Pull request by Rad0van**
This is simple port of purchase_receptions_status to 13.0

### CODE REVIEW

**Review by alexis-via — APPROVED** [review]

**Comment by Rad0van** [comment]
@pedrobaeza would you please care to take a look? Thank you.

**Comment by pedrobaeza** [comment]
Sorry, but I'm not a qualified reviewer of this module.

**Comment by Rad0van** [comment]
Maybe @rvalyi or @quentinDupont could review this please? It's just straight port to 13.0 Thanx

**Comment by kayue** [comment]
Please considering my version of `purchase_order.xml`, it will add the Reception Status column to both RfQ and Purchase Order screen. The existing code only modify the RfQ screen.

https://github.com/OCA/purchase-workflow/blob/bc41abbf0b98382b7325effed116ade2cd6aa7b6/purchase_reception_status/views/purchase_order.xml

**Comment by Rad0van** [comment]
> Please considering my version of `purchase_order.xml`, it will add the Reception Status column to both RfQ and Purchase Order screen. The existing code only modify the RfQ screen.
> 
> https://github.com/OCA/purchase-workflow/blob/bc41abbf0b98382b7325effed116ade2cd6aa7b6/purchase_reception_status/views/purchase_order.xml

@kayue not sure what you mean. I guess it would be best to wait till this is merged and then create PR with improvement. Or create the improvement on 12.0 branch. If that gets merged I will update it here.

**Comment by Rad0van** [comment]
@alexis-via can you please recommend someone to review this? Thank you.

**Comment by kayue** [comment]
Both RfQ and PO screen could show the reception state. 
Right now it is only showing in RfQ

Purchase_Orders_-_Odoo

**Comment by Rad0van** [comment]
> Both RfQ and PO screen could show the reception state.
> Right now it is only showing in RfQ
> 
> Purchase_Orders_-_Odoo

I see. The best thing in my opinion would be what I suggested in my last comment: create a PR on 12.0 branch with improvement. If that gets approved I will incorporate it here.

**Comment by kayue** [comment]
I can wait until this get merged to 13 :)

I couldn't test my changes in 12.0 so I don't feel confident submitting a pull request. (In fact the XML file in 12.0 doesn't work in Odoo 13)

File: purchase_reception_status/views/purchase_order.xml
Comment (kayue) [inline purchase_reception_status/views/purchase_order.xml:48]: I think this filter `not_invoiced` no longer exist in Odoo 13, this yield error for me.

**Review by kayue — CHANGES_REQUESTED** [review]

File: purchase_reception_status/views/purchase_order.xml
Comment (Rad0van) [reply purchase_reception_status/views/purchase_order.xml:48]: Well it is there for me:
image

image

Comment (kayue) [reply purchase_reception_status/views/purchase_order.xml:48]: You are right.

**Review by CasVissers-360ERP — CHANGES_REQUESTED** [review]
Same feedback here, status is missing in purchase_order_view_tree.

**Comment by Rad0van** [comment]
> Same feedback here, status is missing in purchase_order_view_tree.

@CasVissers , you mean the same feedback as kayue? Somehow I cannot see the changes you suggest.

**Comment by HviorForgeFlow** [comment]
@Rad0van have you try this module? https://github.com/OCA/purchase-workflow/blob/13.0/purchase_open_qty
Feature is quite similar

**Comment by Rad0van** [comment]
> @Rad0van have you try this module? https://github.com/OCA/purchase-workflow/blob/13.0/purchase_open_qty
> Feature is quite similar

@HviorForgeFlow no I didn't. Frankly I missed that one. I liked this one but needed it for 13.0 so I tried to do the migration.Both exist in 12.0 :-)

**Comment by Rad0van** [comment]
> Same feedback here, status is missing in purchase_order_view_tree.

@CasVissers-360ERP can you please be more specific? Which file? I would like to finish this one. Thanx.
Edit: I think I know. Working on it.

**Comment by Rad0van** [comment]
@CasVissers-360ERP I guess it should be OK now. Please check. Thank you.

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM 👍

**Comment by Rad0van** [comment]
@CasVissers-360ERP or @alexis-via can you please ask ocabot to merge? Or can I do that? Edit: I cannot but that's expected...

**Comment by HviorForgeFlow** [comment]
@Rad0van can you squash commits, let's move it forward

**Comment by Rad0van** [comment]
> @Rad0van can you squash commits, let's move it forward

Done.

**Pull request merged**
